modules/serverless-overriding-pdbs-eventing.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * /serverless/eventing/tuning/serverless-ha.adoc
+
+
+:_content-type: PROCEDURE
+[id="serverless-overriding-poddistruptionbudgets-eventing_{context}"]
+= Overriding podDisruptionBudgets
```

Comment:
I would probably go just with "Overriding disruption budgets". Seems more appropriate for a heading. Using camel case seems like it already points to some specific configuration element.